operators/pkg/bastion-controller/bastion_controller.go

```diff
@@ -31,16 +32,16 @@ import (
 	crownlabsalpha1 "github.com/netgroup-polito/CrownLabs/operators/api/v1alpha1"
 )
 
-// BastionReconciler reconciles a Bastion object
+// BastionReconciler reconciles a Bastion object.
 type BastionReconciler struct {
 	client.Client
 	Log                logr.Logger
 	Scheme             *runtime.Scheme
 	AuthorizedKeysPath string
 }
 
+// Reconcile reconciles the SSH keys of a Tenant resource.
 // +kubebuilder:rbac:groups=crownlabs.polito.it,resources=tenants,verbs=get;watch
```

Comment:
I think we don't use these markers in our deploy approach, IMHO this can be the right PR to remove them from all the controllers

Reply:
Definitely.